third_party.md

```diff
@@ -1,4 +1,8 @@
-### SDL ANDROID
+### Introduction
+
+This file is broken into three Parts: SDL Base, SDL Android and SDL Java. SDL Android and SDL Java are sub projects of SDL Base. Additionally, SDL Java contains two sub-projects, SDL JavaSE and SDL JavaEE.
+
+## SDL BASE
 
 Copyright (C) 2018 SmartDeviceLink Consortium, Inc.
```

Comment:
Is SDL BASE just now being introduced?  If so, I think this should be `Copyright (C) 2019`.  If not, this should be `Copyright (C) 2018 - 2019`